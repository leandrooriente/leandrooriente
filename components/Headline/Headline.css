.headline {
  min-height: 50vh;
  max-width: 850px;
  display: flex;
  align-items: center;
}

.headline a {
  color: var(--highlight-color);
  border-bottom: 2px solid var(--highlight-color);
  text-decoration: none;
  padding: 0 2px;
}

.headline a:hover {
  color: #FFF;
  background: var(--highlight-color);
  border-bottom-color: transparent;
  transition: all var(--fast-transition) ease-in;
}
