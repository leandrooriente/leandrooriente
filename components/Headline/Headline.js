import React from 'react';
import css from './Headline.css';

const Headline = () => (
  <section className={css.headline}>
    <h1 className={css.title}>
      Hi, Iâ€˜m Leandro Oriente, a front end developer at <a href="https://www.travix.com/">Travix</a> in Amsterdam.
    </h1>
  </section>
);

export default Headline;
