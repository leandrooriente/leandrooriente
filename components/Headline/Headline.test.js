import React from 'react';
import { mount } from 'enzyme';
import Headline from './Headline';

describe('<Headline />', () => {
  it('should render <Header /> component', () => {
    const wrapper = mount(<Headline />);

    expect(wrapper).toMatchSnapshot();
  });
});
