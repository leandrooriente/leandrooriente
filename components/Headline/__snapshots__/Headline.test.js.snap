// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Headline /> should render <Headline /> component 1`] = `
<Headline>
  <section
    className="headline"
  >
    <Title
      className={null}
    >
      <h1
        className="title"
      >
        Hi, Iâ€˜m Leandro Oriente, a front end developer at 
        <a
          href="https://www.travix.com/"
        >
          Travix
        </a>
         in Amsterdam.
      </h1>
    </Title>
  </section>
</Headline>
`;
